packages/ERTP/package.json

```diff
@@ -57,7 +57,13 @@
     "exported.js"
   ],
   "ava-xs": {
-    "exclude": "swingsetTests"
+    "exclude": [
+      "swingsetTests",
+      "# fast-check unsupported",
```

Comment:
do we need to test these in XS?

If not, maybe we have a different directory for tests that run under both Ava and XS?

Reply:
It's not clear to me why fast-check is not supported. What am I missing?

Reply:
They don't _all_ fail but it looked like something with fast-check,
```
. test/unitTests/test-amountProperties.js store
. test/unitTests/test-amountProperties.js reject promise keys
. test/unitTests/test-amountProperties.js passability of stores
. test/unitTests/test-amountProperties.js passability of store iters
. test/unitTests/test-amountProperties.js iteration fails with concurrent addition
. test/unitTests/test-amountProperties.js iteration succeeds with concurrent deletion
. test/unitTests/test-amountProperties.js provide for mapStores
FAIL (todo route console) (TypeError#1)
TypeError#1: ?: no function

{
  status: 'not ok',
  id: 160,
  message: 'isEqual is a (total) equivalence relation threw: ?: no function',
  filename: 'test/unitTests/test-amountProperties.js',
  label: ''
}
F test/unitTests/test-amountProperties.js isEqual is a (total) equivalence relation
FAIL (todo route console) (TypeError#2)
TypeError#2: ?: no function

{
  status: 'not ok',
  id: 161,
  message: 'isGTE is a partial order with empty as minimum threw: ?: no function',
  filename: 'test/unitTests/test-amountProperties.js',
  label: ''
}
F test/unitTests/test-amountProperties.js isGTE is a partial order with empty as minimum
FAIL (todo route console) (TypeError#3)
TypeError#3: ?: no function

{
  status: 'not ok',
  id: 162,
  message: 'add: closed, commutative, associative, monotonic, with empty identity threw: ?: no function',
  filename: 'test/unitTests/test-amountProperties.js',
  label: ''
}
F test/unitTests/test-amountProperties.js add: closed, commutative, associative, monotonic, with empty identity
FAIL (todo route console) (TypeError#4)
TypeError#4: ?: no function

{
  status: 'not ok',
  id: 163,
  message: 'subtract: (x + y) - y = x; (y - x) + x = y if y >= x threw: ?: no function',
  filename: 'test/unitTests/test-amountProperties.js',
  label: ''
}
F test/unitTests/test-amountProperties.js subtract: (x + y) - y = x; (y - x) + x = y if y >= x
FAIL (todo route console) (TypeError#5)
TypeError#5: ?: no function

{
  status: 'not ok',
  id: 164,
  message: 'minmax threw: ?: no function',
  filename: 'test/unitTests/test-amountProperties.js',
  label: ''
}
```

Reply:
ok... fast-check is unsupported in that tests that don't use it don't seem to work and we're not investing in whatever it takes to get them to work yet. fair enough.

for future reference, now that `getStackString()` is working, that's where I would pick up on "todo route console".